docs/index.rst

```diff
@@ -0,0 +1,1032 @@
+BigML Python Bindings
+=====================
+
+`BigML <https://bigml.com>`_ makes machine learning easy by taking care
+of the details required to add data-driven decisions and predictive
+power to your company. Unlike other machine learning services, BigML
+creates
+`beautiful predictive models <https://bigml.com/gallery/models>`_ that
+can be easily understood and interacted with.
+
+These BigML Java bindings allow you to interact with BigML.io, the API
+for BigML. You can use it to easily create, retrieve, list, update, and
+delete BigML resources (i.e., sources, datasets, models and,
+predictions).
+
+This module is licensed under the `Apache License, Version
+2.0 <http://www.apache.org/licenses/LICENSE-2.0.html>`_.
+
+Support
+-------
+
+Please report problems and bugs to our `BigML Java Binding
+issue tracker <https://github.com/bigmlcom/bigml-java/issues>`_.
+
+Discussions about the different bindings take place in the general
+`BigML mailing list <http://groups.google.com/group/bigml>`_. Or join us
```

Comment:
this link to the Google's mailing list I believe that should be removed. That list looks pretty abandoned. It is linked as well in our /api/support section but it will be removed soon, it is also linked from bigmlcom/io though. what do you think @mmerce?